.vscode/launch.json

```diff
@@ -4,7 +4,7 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
-
+    
```

Comment:
This is why we can't have nice things.

(I'm kidding, just found it funny)

Reply:
Not really sure what happened here. We'll assume VS Code did it